### ISSUE TITLE
Unable to update

### ISSUE THREAD

**Issue by JRodriguez556** [bitwarden/self-host#78]
it looks like the run.sh file called by bitwarden.sh is broken.

RUN_SCRIPT_URL="https://func.bitwarden.com/api/dl/?app=self-host&platform=linux&variant=run"

This file seems to be an error message.

**Comment by npodbielski** [comment]
Yes, I have encoutered this too. My cron rund update every day and today Bitwarden did not get back up. Whole run.sh script got replaced by error.

Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonTextReader.Read()
   at Newtonsoft.Json.Linq.JObject.Load(JsonReader reader, JsonLoadSettings settings)
   at Newtonsoft.Json.Linq.JObject.Parse(String json, JsonLoadSettings settings)
   at AppFunctions.Downloads.GetToolsDownloadUrlAsync(String repository, String filename) in /home/runner/work/misc/misc/AzureFunctions/prod/AppFunctions/Downloads.cs:line 240
   at AppFunctions.Downloads.GetDownloadUrlAsync(ValueTuple`4 valueTuple) in /home/runner/work/misc/misc/AzureFunctions/prod/AppFunctions/Downloads.cs:line 197
   at AppFunctions.Downloads.Run(HttpRequest req, ILogger log) in /home/runner/work/misc/misc/AzureFunctions/prod/AppFunctions/Downloads.cs:line 174
